frontend/package.json

```diff
@@ -77,8 +77,8 @@
     "compile-scss": "sass --load-path=./node_modules/uswds/dist/scss --style=compressed --no-source-map src/scss/App.scss:src/styles/App.css",
     "watch-css": "yarn compile-scss && sass --load-path=./node_modules/uswds/dist/scss --style=compressed --no-source-map -w src/scss/App.scss:src/styles/App.css",
     "create-storybook-public": "rm -rf ./storybook_public && cp -R ./public ./storybook_public && cp mockServiceWorker.js ./storybook_public",
-    "storybook": "yarn create-storybook-public && yarn watch-css & start-storybook -p 6006 -s storybook_public",
-    "build-storybook": "yarn create-storybook-public && yarn compile-scss && REACT_APP_BACKEND_URL=http://localhost:8080 build-storybook -s storybook_public",
+    "storybook": "yarn create-storybook-public && yarn watch-css & storybook dev -p 6006 -s storybook_public",
```

Comment:
```suggestion
    "storybook": "yarn create-storybook-public && yarn watch-css & storybook dev -p 6006 -s ../storybook_public",
```
@alismx I poked around for the issue Elisa raised and found [this GitHub issue](http://localhost:6006/?path=/docs/example-introduction--docs) and [this docs page](https://storybook.js.org/addons/msw-storybook-addon). I think the issue is that our yarn script here is pointing at the wrong filepath for the `storybook_public` directory where the MSW setup script was housed and so on startup, storybook can't find it. 

Changing this to the static dirs filepath to `../storybook_public` seems to have fixed the issue locally for me

<img width="1328" alt="Screenshot 2023-07-28 at 12 13 17 PM" src="https://github.com/CDCgov/prime-simplereport/assets/29645040/39877a00-32dd-42f3-99ce-bbdfaecc09af">

Reply:
Thanks so much! 🙌 

I thought something was wrong with msw or our worker file. I'll push your suggestion up asap.